local_tests/test_models.py

```diff
@@ -2,16 +2,20 @@
 import io
```

Comment:
this filename sounds like it's testing the models. Should it be renamed to test_classes or similar? But I guess that could be confusing also

Reply:
I'll leave it for now but I agree there could be  a better name for this and other files in this folder